tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -198,3 +198,26 @@ def test_positive_search_in_org(self):
             {lc_env_.name for lc_env_ in lc_envs},
             {u'Library', lc_env.name},
         )
+
+    @tier2
+    @skip_if_bug_open('bugzilla', 1348727)
+    def test_verify_bugzilla_1348727(self):
```

Comment:
@JacobCallahan is that a namig pattern we addopt instead of 'test_positive_create_environment_after_host_register' ?